### PR TITLE
Add player component with simple styles

### PR DESCRIPTION
**Pull request by macfrei**
Player component with simple styling.

### CODE REVIEW

File: src/components/Player.css [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+}
+
+.Player__score {
+  display: grid;
+  gap: 5px;
+  grid-template-columns: repeat(3, 1fr);
+  place-items: center;
+}
+
+.Player__score > button {
```
Comment (badabam) [inline src/components/Player.css]:
```suggestion
.Player__button {
```

Comment (badabam) [reply src/components/Player.css]: We could also give all buttons in `index.css` a minimal default padding. Or search for a base-css file. I am thinking about a small css library that does what we need. I'll look it up.

File: src/components/Player.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import './Player.css'
+
+export default function Player({ name, score, onMinus, onPlus }) {
+  return (
+    <section className="Player">
+      <span>{name}</span>
```
Comment (badabam) [inline src/components/Player.js]: Not needed, I guess

**Review by badabam — APPROVED** [review]

File: src/components/Player.js (hunk 2 above)
Comment (macfrei) [reply src/components/Player.js]: True 😄

**Pull request merged**